pdns/dnspacket.cc

```diff
@@ -101,6 +103,8 @@ DNSPacket::DNSPacket(const DNSPacket &orig)
   d_eso = orig.d_eso;
   d_haveednssubnet = orig.d_haveednssubnet;
   d_haveednssection = orig.d_haveednssection;
+  d_ednsversion = orig.d_ednsversion;
+  d_ednsrcode = 0;
```

Comment:
if we actually used copy ctor for it, then yes. we don't do it, but I'll add it just to be explicit about it.